src/sidebar/services/groups.js

```diff
@@ -106,14 +107,22 @@ export default function groups(
         directLinkedGroupId = null;
       }
     }
-
     // If service groups are specified only return those.
     // If a service group doesn't exist in the list of groups don't return it.
     if (svc && svc.groups) {
-      const focusedGroups = groups.filter(
-        g => svc.groups.includes(g.id) || svc.groups.includes(g.groupid)
-      );
-      return focusedGroups;
+      try {
+        // The groups may not be available if they are being fetched via RPC.
+        // See mutateAsyncGroups in fetch-config.js
```

Comment:
The reference to `mutateAsyncGroups` is obsolete. I would just shorten this to "See fetch-config.js" and avoid mentioning implementation details as that is less likely to get out of date.